### DISCUSSION TITLE
Is there a callback for when we click on one of the navigation buttons?

### DISCUSSION THREAD

**Discussion by WagyuDev** [wojtekmaj/react-calendar#605] [Q&A] [answered]
I need to do something when a user clicks on the next/prev buttons. Is there any props I can use? something like "onNavigate" or "onMonthChange"

**Comment by wojtekmaj** [comment, marked as answer]
Navigation button click always result in `activeStartDate` change. So you use `onActiveStartDateChange`, which gets called with an object with `action` property, will tell you which button caused active start date to change, in a format documented in README.
